libraries/Ethernet/examples/ETH_W5500_IDF_SPI/ETH_W5500_IDF_SPI.ino

```diff
@@ -94,7 +94,7 @@ void setup()
 {
   Serial.begin(115200);
   Network.onEvent(onEvent);
-  ETH.begin(ETH_TYPE, ETH_ADDR, ETH_CS, ETH_IRQ, ETH_RST, ETH_SPI_HOST, ETH_SPI_SCK, ETH_SPI_MISO, ETH_SPI_MOSI);
+  ETH.begin(ETH_PHY_TYPE, ETH_PHY_ADDR, ETH_PHY_CS, ETH_PHY_IRQ, ETH_PHY_RST, ETH_PHY_SPI_HOST, ETH_PHY_SPI_SCK, ETH_PHY_SPI_MISO, ETH_PHY_SPI_MOSI);
```

Comment:
From experience, I can tell you that using the other `begin()` function which accepts a SPI host instance as argument is way easier to use.
Especially when you also intend to use the same SPI bus for something else like a display.
So can you at least add some comment to this example advising to use the one with a SPI host argument if they intend to use the bus for something else?

That will for sure save quite a lot of hours for those running into this.

Reply:
there are two examples. One with IDF SPI and one with Arduino SPI